pyperformance/_manifest.py

```diff
@@ -170,14 +170,12 @@ def _add_benchmark(self, spec, metafile, resolve, filename):
     def _add_group(self, name, entries):
         if name in self._byname:
             raise ValueError(f'a group and a benchmark have the same name ({name})')
-        if name == 'all':
-            raise ValueError('a group named "all" is not allowed ("all" is reserved for selecting the full set of declared benchmarks)')
         if entries is None:
             if name in self._raw_groups:
                 return
             self._raw_groups[name] = None
         elif name in self._raw_groups and self._raw_groups[name] is not None:
-            raise ValueError(f'a group named {name} was already defined')
+            self._raw_groups[name].extend(list(entries) if entries else [])
```

Comment:
This rule should apply for a single manifest, with the exception only for includes (which should be merged like you are doing).  I expect the logic for distinguishing between the two cases is lacking, either in `_parse_manifest() below or in the code that consumes its output.